tests/test_http_stats.py

```diff
@@ -0,0 +1,91 @@
+"""Validate that the worker id in the context of the http proxy is the id of the worker rather than the queue."""
+
+import unittest
+from http.server import HTTPServer, BaseHTTPRequestHandler
+
+from saq import Queue, Worker
+from saq.queue.http import HttpProxy
+from saq.types import Context
+from tests.helpers import setup_postgres, create_postgres_queue, teardown_postgres
+import asyncio
+import threading
+
+
+async def echo(_ctx: Context, *, a: int) -> int:
+    return a
+
+
+class ProxyRequestHandler(BaseHTTPRequestHandler):
+    def __init__(self, *args, proxy=None, **kwargs):
+        self.proxy = proxy
+        super().__init__(*args, **kwargs)
+
+    def do_POST(self):
```

Comment:
nit all lower

Reply:
not relevant anymore as moved to aiohttp